### PR TITLE
chore(install): convert stdout messages to a reporter implementation

### PR DESCRIPTION
**Pull request by zlesnr**

### CODE REVIEW

**Review by sanderblue — DISMISSED** [review]
🐑 it

File: internal/install/recipe_installer.go [hunk 1]
```diff
@@ -259,18 +243,20 @@ func (i *RecipeInstaller) installRecipesWithPrompts(m *types.DiscoveryManifest,
 		}
 
 		log.Debugf("Executing and validating with progress for recipe name %s...", r.Name)
+
+		// TODO Handler the error here.  The tests indicate that this method should
```
Comment (ctrombley) [inline internal/install/recipe_installer.go:247]: Do we really need to do anything else here?  We are handling the error currently, by swallowing it & logging.

**Review by ctrombley — APPROVED** [review]
🚢

File: internal/install/recipe_installer.go (hunk 1 above)
Comment (zlesnr) [reply internal/install/recipe_installer.go:247]: Oh yeah, this has been resolved.

**Review by sanderblue — APPROVED** [review]

**Review by ctrombley — APPROVED** [review]

**Pull request merged**